unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/SettingsControllers/Tests/SettingsControlsShould.cs

```diff
@@ -1,14 +1,15 @@
-using System.Collections.Generic;
-using System.Reflection;
 using Cinemachine;
+using DCL.Camera;
 using DCL.Rendering;
 using DCL.SettingsCommon.SettingsControllers.BaseControllers;
 using DCL.SettingsCommon.SettingsControllers.SpecificControllers;
 using NUnit.Framework;
+using System.Collections.Generic;
+using System.Reflection;
```

Comment:
New imports added, are all of them necessary?

Reply:
Not really, I was fixing test and they just lingered there. Removing